be/src/http/action/update_config_action.cpp

```diff
@@ -64,11 +65,13 @@ void UpdateConfigAction::handle(HttpRequest* req) {
         s = config::set_config(config, new_value);
         if (s.ok()) {
             LOG(INFO) << "set_config " << config << "=" << new_value << " success";
-
             if (_config_callback.count(config)) {
                 _config_callback[config]();
             }
-
+            if (config == "storage_page_cache_limit") {
```

Comment:
use unified callback to implement real update logic.

Reply:
have been solved